channels/eus-4.6.yaml

```diff
@@ -0,0 +1,5 @@
+name: eus-4.6
```

Comment:
`extended-update-support-4.6`?  Or are there benefits to squeezing this down to fewer characters?

Reply:
I'm not opposed to that but "eus-4.6" is what I'd proposed and everyone has agreed so far. I'm sure such a long string would wrap in the current console UX and I see this more as a channel id than a full description. We should get those descriptive channel text api in :-)